onnxruntime/core/graph/contrib_ops/bert_defs.cc

```diff
@@ -1110,12 +1110,12 @@ ONNX_MS_OPERATOR_SET_SCHEMA(
                OpSchema::Optional)
         .Input(5,
                "seqlens_k",
-               // For prompt, the value is number of tokens (excluding padding) - 1.
-               "1d Tensor of shape (batch_size). Indicates past sequence lengths for token generation case.",
+               "1D Tensor of shape (batch_size). Equivalent to (total_sequence_lengths - 1).",
```

Comment:
I saw PR description mention "use of a 2-D seqlens_k tensor". Any change is needed here?

Reply:
I will change the description, that was an old idea